docs/source/building-applications.rst

```diff
@@ -4,6 +4,12 @@
 Building Applications
 =====================
 
+The Building Applications section covers the elements that are used to create, extend, and test your Daml full-stack application (including APIs and script libraries) and the architectural best practices for bringing those elements together.  
```

Comment:
What are the "script libraries" you mention?

Reply:
The JavaScript Client libraries - would you prefer I specify?